manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -4980,7 +4982,7 @@ metadata:
   name: istiod-istio-system
   labels:
     app: istiod
-    release: istio-base
+    release: istio
```

Comment:
Why this change?

Reply:
This is for kustomize - changing it to be the same as istioctl for consistent experience